pljava-packaging/src/main/java/Node.java

```diff
@@ -15,97 +15,40 @@
-
-/*
- * For "Node" behavior:
- */
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 import static java.lang.ProcessBuilder.Redirect.INHERIT;
-import java.lang.reflect.InvocationHandler; // flexible SAM allowing exceptions
+import static java.lang.System.getProperty;
+import static java.lang.System.setProperty;
 import static java.lang.Thread.interrupted;
-
 import static java.net.InetAddress.getLoopbackAddress;
 import static java.net.URLEncoder.encode;
-import java.net.ServerSocket;
-
+import static java.nio.charset.Charset.defaultCharset;
 import static java.nio.charset.StandardCharsets.US_ASCII;
-
-import static java.nio.file.Files.createTempFile;
-import static java.nio.file.Files.createTempDirectory;
-import static java.nio.file.Files.deleteIfExists;
-import static java.nio.file.Files.exists;
-import static java.nio.file.Files.getLastModifiedTime;
-import static java.nio.file.Files.lines;
-import static java.nio.file.Files.walk;
-import static java.nio.file.Files.write;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import static java.nio.file.Files.*;
 import static java.nio.file.StandardWatchEventKinds.*;
-import java.nio.file.WatchEvent;
-import java.nio.file.WatchKey;
-import java.nio.file.WatchService;
-
-import java.nio.file.AccessDeniedException;
-import java.nio.file.NoSuchFileException;
-
-import java.sql.Connection;
 import static java.sql.DriverManager.getConnection;
-import java.sql.ParameterMetaData;
-import java.sql.PreparedStatement;
-import java.sql.Statement;
-import java.sql.ResultSet;
-import java.sql.ResultSetMetaData;
-import java.sql.Types;
-
-import java.sql.SQLException;
-import java.sql.SQLWarning;
-
-import javax.sql.rowset.RowSetProvider;
-import javax.sql.rowset.WebRowSet;
-import javax.sql.rowset.RowSetMetaDataImpl;
-
-import java.util.ArrayDeque;
-import java.util.Base64;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Map;
-import java.util.Objects;
 import static java.util.Objects.requireNonNull;
-import java.util.Properties;
-import java.util.Random;
-import java.util.Spliterator;
-import static java.util.Spliterator.IMMUTABLE;
-import static java.util.Spliterator.NONNULL;
-import static java.util.Spliterator.ORDERED;
+import static java.util.Spliterator.*;
 import static java.util.Spliterators.spliteratorUnknownSize;
-
-import java.util.concurrent.Callable; // like a Supplier but allows exceptions!
-import java.util.concurrent.CancellationException;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
-
-import java.util.function.BooleanSupplier;
-import java.util.function.Consumer;
-import java.util.function.Function;
-import java.util.function.Predicate;
-import java.util.function.Supplier;
-import java.util.function.UnaryOperator;
-
-import java.util.jar.JarFile;
-
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
+import static java.util.regex.Pattern.compile;
```

Comment:
Could you please undo all of these changes to the imports, and find whatever setting in your IDE has to be changed so it stops doing that? It is especially irksome when it moves around imports that were arranged with comments, and loses the comments.

I would like this commit to show zero changes in this file except for the one added comment about the duplicated code.